modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -217,44 +261,37 @@ impl RoutingInfo {
     }
 }
 
-#[derive(
-    Debug, Clone, Eq, PartialEq, PartialOrd, Hash, Serialize, Deserialize, Decodable, Encodable,
-)]
+#[derive(Debug, Clone, Eq, PartialEq, PartialOrd, Hash, Serialize, Deserialize)]
 pub struct PaymentFee {
     pub base: Amount,
     pub parts_per_million: u64,
 }
 
-const DUST_LIMIT: Amount = Amount::from_sats(50);
-
 impl PaymentFee {
-    pub const SEND_FEE_MINIMUM: PaymentFee = PaymentFee {
-        base: DUST_LIMIT,
-        parts_per_million: 5_000,
-    };
-
-    // The difference between the SEND_FEE_LIMIT_DEFAULT and the SEND_FEE_MINIMUM
-    // leaves the gateway with a limit of one percent + 50 sats for lightning
-    // routing fees which is the LDK default
+    /// This is the maximum send fee of one and a half percent plus one hundred
+    /// satoshis a correct gateway may recommend as a default. It accounts for
+    /// the fee required to reliably route this payment over lightning.
     pub const SEND_FEE_LIMIT_DEFAULT: PaymentFee = PaymentFee {
         base: Amount::from_sats(100),
         parts_per_million: 15_000,
     };
 
+    /// This is the maximum receive fee of half of one percent plus fifty
+    /// satoshis a correct gateway may recommend as a default.
     pub const RECEIVE_FEE_LIMIT_DEFAULT: PaymentFee = PaymentFee {
-        base: DUST_LIMIT,
+        base: Amount::from_sats(50),
         parts_per_million: 5_000,
     };
```

Comment:
I assume all these defaults are adjustable? Seem way too high, but not worth discussing if operators can just lower them.

Reply:
operators can adjust them. its just a client side limit.